jest-environment-jsdom.cjs

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+const { TextEncoder, TextDecoder } = require("util");
+const {
+  default: $JSDOMEnvironment,
+  TestEnvironment,
+} = require("jest-environment-jsdom");
+
+Object.defineProperty(exports, "__esModule", {
+  value: true,
+});
+
+// Using Jest 29, TextEncoder and TextDecoder do not have support, see https://github.com/jsdom/jsdom/issues/2524,
```

Comment:
Is it most-likely to be resolved that Jest will have support for TextEncoder/TextDecoder?

Reply:
And will we find out?